### PR TITLE
chore: delete IntelliJ run configs

### PR DESCRIPTION
**Pull request by warber**

### CODE REVIEW

File: .run/shared/Mocks.run.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<component name="ProjectRunConfigurationManager">
```
Comment (UnseenWizzard) [inline .run/shared/Mocks.run.xml:1]: did you include this on purpose?

File: e2e.env [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# This env file can be used to provide environment variables for running e2e tests
```
Comment (UnseenWizzard) [inline e2e.env:1]: Could you please: 
1. include this in gitignore, I'd like to safeguard against accidentally checking in a file filled with secrets.
2. mention this in CONTRIBUTING.md

And it's also helpful without an IDE, by just filling it out and sourcing it before running make integration-test, isn't it? Worth mentioning IMO

Comment (warber) [reply e2e.env:1]: ad 1) yes good point
ad 2) ok

ad bonus q) i think you would need to export them if you want to use them with the `source`command

File: .run/shared/Mocks.run.xml (hunk 1 above)
Comment (warber) [reply .run/shared/Mocks.run.xml:1]: i think intellij did that

**Review by Laubi — APPROVED** [review]

**Pull request merged**